ecs-agent/tmds/handlers/fault/v1/handlers/handlers.go

```diff
@@ -39,7 +39,7 @@ const (
 	stopFaultRequestType        = "stop %s"
 	checkStatusFaultRequestType = "check status %s"
 	invalidNetworkModeError     = "%s mode is not supported. Please use either host or awsvpc mode."
-	faultInjectionEnabledError  = "fault injection is not enabled for task: %s"
+	faultInjectionEnabledError  = "enableFaultInjection is not enabled for task: %s"
```

Comment:
nit: I personally prefer the older error message as it is not coupled to the name of the task definition parameter but describes the issue in general. No strong opinion though.

Reply:
I don't have a strong preference as well. I like the new one a bit more because if customers see this message they can know what changes need to be done in their tasks.